### PR TITLE
docs: refactored readme file

### PR DESCRIPTION
**Pull request by Eddievin**
Refactored, edited and added styling to the readme file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,30 @@
-# Solution_Provider
+# Solution_Provider 🍀
 
+- **Supported Tools** - `pip`
+- **Supported Python Version** - `Python 3.7, 3.8 pip`
```
Comment (SanthoshS20) [inline README.md:4]: Can you remove pip and add like this **Python 3.7, Python 3.8**

Comment (Eddievin) [reply README.md:4]: okay.

Comment (Eddievin) [reply README.md]: done, please review.

Comment (SanthoshS20) [reply README.md]: @Eddievin Please take a look on the other repos. If you find anything that you can able to do it. Start working on it. For me this not mandatory to assign a task. Who comes first with a PR, I will accept. That's it

Thanks

**Pull request merged**